niworkflows/reports/core.py

```diff
@@ -16,6 +16,12 @@
 import jinja2
 from nipype.utils.filemanip import copyfile
 
+# Add a new figures spec
+try:
+    add_config_paths(figures=pkgrf('niworkflows', 'reports/figures.json'))
+except ValueError as e:
+    if "Configuration 'figures' already exists" != str(e):
+        raise
```

Comment:
This is pretty cheap to add in general, and putting it inside an `__init__` made this file a bit brittle with `xdist`.